section/ontology.md

```diff
@@ -413,3 +413,238 @@ ex:leftPadMapping
     fno:returnMapping    [ a                   fnom:DefaultReturnMapping ;
                            fnom:functionOutput ex:outputStringOutput ] .
 ```
+
+### Function Composition {#composition}
+
+
```

Comment:
I'd add a `> Function Composition is experimental and subject to change` disclaimer here

Reply:
done - I think. Maybe check if what I did is what you wanted.